README.md

```diff
@@ -41,3 +41,10 @@ Ibis currently provides tools for interacting with the following systems:
 - [Spark](https://spark.apache.org) (Experimental)
 
 Learn more about using the library at http://docs.ibis-project.org.
+
+
+## Notes
+
```

Comment:
can we not fix to a specific image? latest is a moving target which is not great

Reply:
Just choose the latest hash of the dev image and use that and document it here?

Reply:
yes as long as it’s a fixed image then ok

Reply:
I just fixed the tag into docker-compose. related to the note I just added a reference to docker-compose to avoid to have the docker image tag in 2 places.